### ISSUE TITLE
5x5 kernel size instead of 3x3?

### ISSUE THREAD

**Issue by sudo-bcli** [pytorch/tutorials#924]
Hi, I just read this tutorial on your official website [NEURAL NETWORKS](https://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html#sphx-glr-beginner-blitz-neural-networks-tutorial-py)
and think according to the image and the following code, maybe the kernel size of the first convolution layer should be 5x5 instead of 3x3.
If we follow [this formula](https://stackoverflow.com/questions/44193270/how-to-calculate-the-output-size-after-convolving-and-pooling-to-the-input-image) and by default the argument of **conv2d** is **padding = 0** and **stride = 1**, we have
* **1st conv2d with 5x5 kernel**: 32x32 -> 28x28
* **1st max pooling**: 28x28 -> 14x14
* **2nd conv2d with 3x3 kernel**: 14x14 -> 12x12
* **2nd max pooling**: 12x12 -> 6x6

Which will explain both the image and the following linear layer (6x6 image dimension) in your code.

**Comment by holly1238** [comment]
Kernel size is 5x5. This was fixed in the tutorial in https://github.com/pytorch/tutorials/pull/1257.
